README.md

```diff
@@ -266,36 +278,41 @@ Reference: https://stripe.com/docs/upgrades
 
 ## X-Headers
 
-The use of `X-Custom-Header` has been deprecated, see: http://tools.ietf.org/html/rfc6648
+The use of `X-Custom-Header` has [been deprecated](http://tools.ietf.org/html/rfc6648).
 
 ## Resource filtering
-Resource filters should always be in singular form.
+Resource filters MUST be in singular form.
 
-Multiple ids supplied to one filter, as a comma separated list, should be translated into an `OR` query. Chaining multiple filters with `&` should be translated into an `AND` query.
+Multiple ids should be supplied to a filter as a comma separated list, and should be translated into an `OR` query. Chaining multiple filters with `&` should be translated into an `AND` query.
 
-#### Good:
-```
+#### Good
+
+```http
 GET /refunds?payment=ID1,ID2&customer=ID1
 ```
 
-#### Bad:
-```
+#### Bad
+
+```http
 GET /refunds?payments=ID1,ID2&customer=ID1
 ```
 
 ## Pagination
 
 All list/index endpoints must be paginated by default.
-Pagination must be reverse chronological. To enable cursor based pagination, ids should be increasing.
+
+Pagination must be reverse chronological. Ids should increase to enable cursor based pagination.
```

Comment:
could :hocho: ` Ids should increase to enable cursor based pagination.`, related to implementation and only one way of doing it